frontend/src/utils/theme-changer.ts

```diff
@@ -0,0 +1,10 @@
+const useThemeClass = () => {
+  const storedTheme = localStorage.getItem('theme');
+  const prefersDark = window.matchMedia?.('(prefers-color-scheme: dark)').matches;
+  const initialTheme = storedTheme === 'dark' || (!storedTheme && prefersDark);
+  initialTheme && document.documentElement.classList.add('dark');
+  if(initialTheme) return('dark')
+  else return('light')
+};
```

Comment:
please run `npm run format` in the frontend

Reply:
ok sir